lib/lotus/model/migrator/sqlite_adapter.rb

```diff
@@ -69,7 +69,7 @@ def load
         # @api private
         def path
           root.join(
-            @connection.uri.sub(/#{ @connection.adapter_scheme }\:\/\//, '')
+            @connection.uri.sub(/(jdbc\:|)sqlite\:\/\//, '')
```

Comment:
I think this is done by sequel gem extracting scheme from url, https://github.com/jeremyevans/sequel/blob/5a6c7e4ea39d5adeeab853f2d359981eadb2fea4/lib/sequel/adapters/jdbc.rb

have you tried testing this with the correct scheme and using `@connection.adapter_scheme` ?